cmd/status.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright © 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+	"syscall"
+
+	"github.com/JAORMX/selinuxd/pkg/daemon"
+	"github.com/olekukonko/tablewriter"
+	"github.com/spf13/cobra"
+)
+
+// statusCmd represents the status command
+var statusCmd = &cobra.Command{
+	Use:   "status [policy]",
+	Args:  cobra.RangeArgs(0, 1),
+	Short: "Get the status of a policy",
+	Long:  `This gets the status of the given policy.`,
+	Run:   statusCmdFunc,
+}
+
+// nolint:gochecknoinits
+func init() {
+	rootCmd.AddCommand(statusCmd)
+	defineStatusFlags(statusCmd)
+}
+
+func defineStatusFlags(rootCmd *cobra.Command) {
+	rootCmd.Flags().String("socket-path", daemon.DefaultUnixSockAddr, "The path to the socket to listen at")
```

Comment:
same comment about the description as above

Reply:
Done.